docs/source/orderer/ordering_service.md

```diff
@@ -74,41 +74,47 @@ process with three phases that ensures all of the peers in a blockchain network
 keep their ledgers consistent with each other.
 
 In the first phase, a client application sends a transaction proposal to
-a subset of peers that will invoke a smart contract to produce a proposed
-ledger update and then endorse the results. The endorsing peers do not apply
-the proposed update to their copy of the ledger at this time. Instead, the
-endorsing peers return a proposal response to the client application. The
+the Fabric Gateway service. The target peer --- a peer in the same organization
+submitting the proposal --- then runs the transaction and forwards the proposal
+to endorsing peers in the organizations required by the endorsement policy. The
+target peer will invoke a smart contract to execute the transaction and return
+the proposed ledger update to the client, along with the endorsement results. The
 endorsed transaction proposals will ultimately be ordered into blocks in phase
-two, and then distributed to all peers for final validation and commit in
-phase three.
+two, and then distributed to all peers for final validation and commitment to
+the ledger in phase three.
+
+Note: Instead of delegating transaction endorsement to the gateway, the client
+application can specify the endorsing organizations and collect the endorsements --- refer to the [v2.3 Applications and Peers](https://hyperledger-fabric.readthedocs.io/en/release-2.3/peers/peers.html#applications-and-peers) topic for details.
```

Comment:
See my comment in peer topic about not putting "can specify the endorsing organizations and collect the endorsements" in the same sentence.

Reply:
"Note: Instead of delegating to the gateway, the client application can manage transaction endorsement by using a Fabric v2.3 SDK." 

This is a more general statement about endorsement - less specific but maybe makes the point more clearly.